docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -1442,12 +1447,11 @@ the following:
            of the compressed stream filled with zeros (if the
            fill bits are not zero, then the stream should be
            rejected as invalid)
-   2 bits: MNIBBLES, # of nibbles to represent the uncompressed
-           length, encoded as follows: if set to 3, MNIBBLES is 0,
-           otherwise MNIBBLES is the value of this field plus 4.
-           If MNIBBLES is 0, the meta-block is empty, i.e. it does
-           not generate any uncompressed data. In this case, the
-           rest of the meta-block has the following format:
+   2 bits: MNIBBLES - 4, # of nibbles to represent the uncompressed
```

Comment:
I think the original wording is a little bit better here (though it is a bit confusing either way). It is analogous to the definition of  e.g. NBLTYPESL, where we describe how the various bit patterns correspond to various values of NBLTYPESL, that is what "encoded as follows" means.

If we want to make it absolutely clear, it should probably be something like this:


MNIBBLES, # of nibbles to represent the uncompressed length, encoded with
           the following fixed length code:

                       Value    Bit Pattern
                        -------    --------------
                           0                    11
                           4                    00
                           5                    01
                           6                    10

If MNIBBLES is 0, the meta-block is empty, i.e. it does
not generate any uncompressed data. In this case, the
rest of the meta-block has the following format: